ndcube/wcs/wrappers/tests/test_resampled_wcs.py

```diff
@@ -17,7 +17,29 @@ def celestial_wcs(request):
     return request.getfixturevalue(request.param)
 
 
-EXPECTED_2D_REPR = """
+EXPECTED_2D_REPR_NUMPY2 = """
```

Comment:
I opened https://github.com/astropy/astropy/issues/16585 to track this.

Reply:
Do you want me to undo this and wait for astropy?

Reply:
There is now an astropy PR, so maybe? :thinking:

Reply:
But we should still need this for oldest deps?